src/database/models.py

```diff
@@ -957,6 +969,35 @@ class Meta:
         db_table = "user_profiles"
         ordering = ("-created_at",)
 
+class PolicyAcceptanceRecords(models.Model):
```

Comment:
@frimpongopoku - there is a model named Policy, which I think we should use to keep the policy which the Admin is agreeing to.  This model should have a link to the version of the policy they agree to.